extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/communicator/client/RpcClient.java

```diff
@@ -391,11 +401,18 @@ private String getRpcAddress(String address) {
 
     /**
      * Returns the address suffix that specifies the channel for a given HmChannel. This is either a colon ":" followed
-     * by the channel number, or the empty string. The empty string is returned for the channel -1, since this channel
-     * encapsulates the ParamSets of a device that do not belong to one of its channels.
+     * by the channel number, or the empty string for a dummy channel.
      */
     private String getChannelSuffix(HmChannel channel) {
-        return channel.getNumber() != -1 ? ":" + channel.getNumber() : "";
+        return isDummyChannel(channel) ? "" : ":" + channel.getNumber();
+    }
+
+    /**
+     * Checks whether a channel is a dummy channel. The dummy channel of a device encapsulates the MASTER Paramset that
+     * does not belong to one of its channels.
+     */
+    private boolean isDummyChannel(HmChannel channel) {
```

Comment:
There is not much information in the term DummyChannel (except that dummy implies there is something hacked). So the explanation is only visible in the javadoc. I would suggest to use the term `MaintenanceChannel` instead. Additionally you may create a constant `MAINTENANCE_CHANNEL_NUMBER` which is used in this method as well as in `ListDevicesParser#parse(Object[] message)`. That way it would be more comprehensible where this channel comes from

Reply:
I like the idea of the constant, but I'm not so sure about the name (although I agree that Dummy is not the best choice). IIRC the HmChannel 0 is mapped to an ESH channel called "Maintenance" for at least some devices, so this name might cause confusion. Since this channel only has a MASTER Paramset, and the Datapoints of MASTER Paramsets are used as configuration parameters, how about calling the channel "ConfigurationChannel"?

Reply:
Sounds good, I didn't think about the fact that the channel Maintenance exists for some device types